tests/Aspire.Templates.Tests/TemplateTestsBase.cs

```diff
@@ -357,7 +357,7 @@ await CheckDashboardHasResourcesAsync(
             // Current SDK, Current TFM
             { templateName, extraArgs, TestSdk.Current, TestTargetFramework.Current, null },
             // Current SDK, Next TFM
-            { templateName, extraArgs, TestSdk.Current, TestTargetFramework.Next, "The current .NET SDK does not support targeting .NET 10.0" },
+            { templateName, extraArgs, TestSdk.Current, TestTargetFramework.Next, null },
```

Comment:
Why not keep the current TFM actually be current (meaning 10.x) and then removing the expected error? (which is what I had) That way we are no validating that with a .NET 10 SDK you can build an aspire app targeting 10.

Reply:
Can you elaborate on what is the missing case here, or the incorrect one?